src/transformers/models/gpt_neox/configuration_gpt_neox.py

```diff
@@ -0,0 +1,125 @@
+# coding=utf-8
+# Copyright 2022 EleutherAI and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" GPTNeoX model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+GPT_NEOX_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "gpt-neox-20b": "https://huggingface.co/gpt-neox-20b/resolve/main/config.json",
+    # See all GPTNeoX models at https://huggingface.co/models?filter=gpt_neox
+}
+
+
+class GPTNeoXConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`~GPTNeoXModel`]. It is used to instantiate an
```

Comment:
```suggestion
    This is the configuration class to store the configuration of a [`~transformers.GPTNeoXModel`]. It is used to instantiate an
```
Not sure if necessary, @sgugger can correct me if not

Reply:
Should just be
```suggestion
    This is the configuration class to store the configuration of a [`GPTNeoXModel`]. It is used to instantiate an
```